components/places/ffi/src/lib.rs

```diff
@@ -10,13 +10,13 @@
 
 use ffi_support::{
     define_box_destructor, define_bytebuffer_destructor, define_handle_map_deleter,
-    define_string_destructor, ByteBuffer, ConcurrentHandleMap, ExternError, FfiStr,
+    define_string_destructor, ConcurrentHandleMap, ExternError, FfiStr,
 };
+use uniffi::ViaFfi;
 use places::error::*;
```

Comment:
It looks like `type ByteBuffer = uniffi::RustBuffer` would help keep the diff down